src/main/java/com/petros/bibernate/session/Session.java

```diff
@@ -57,6 +59,16 @@ public interface Session {
      */
     <T> T find(Class<T> entityClass, Object primaryKey);
 
+    /**
+     * Find by entity class.
+     * Search for all entities of the specified class.
+     * It does not check persistence context but calls database directly.
+     *
+     * @param entityClass entity class
+     * @return the found entity instances or an empty list
+     */
+    <T> List<T> findAll(Class<T> entityClass);
```

Comment:
Are you sure we need such a method in the session? Hibernate does not have a similar method. If it's only needed for testing purposes, would it make more sense to create it as a utility method specifically for tests and inside test package?